docs/snippets/ov_preprocessing_migration.py

```diff
@@ -0,0 +1,110 @@
+# Copyright (C) 2018-2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+from openvino.runtime import Core, Layout, Type
+from openvino.preprocess import PrePostProcessor
+
+#include "inference_engine.hpp"
+
+model_path = ''
+tensor_name = ''
+
+core = Core()
+model = core.read_model(model=model_path)
+
+#! [ov_mean_scale]
+from openvino.runtime import Layout
+ppp = PrePostProcessor(model)
+input = ppp.input(tensor_name)
+# we only need to know where is C dimension
+input.model().set_layout(Layout('...C'))
+# specify scale and mean values, order of operations is important
+input.preprocess().mean([116.78]).scale([57.21, 57.45, 57.73])
+# insert preprocessing operations to the 'model'
+model = ppp.build()
+#! [ov_mean_scale]
+
+#! [ov_conversions]
+from openvino.runtime import Layout, Type
+ppp = PrePostProcessor(model)
+input = ppp.input(tensor_name)
+input.tensor().set_layout(Layout('NCHW')).set_element_type(Type.u8)
+input.model().set_layout(Layout('NCHW'))
+# layout and precision conversion is inserted automatically,
+# because tensor format != model input format
+model = ppp.build()
+#! [ov_conversions]
+
+#! [ov_color_space]
+from openvino.preprocess import ColorFormat
+from openvino.runtime import Layout, Type
+ppp = PrePostProcessor(model)
+input = ppp.input(tensor_name)
+input.tensor().set_color_format(ColorFormat.NV12_TWO_PLANES)
+# add NV12 to BGR conversion
+input.preprocess().convert_color(ColorFormat.BGR)
+# and insert operations to the model
+model = ppp.build()
+#! [ov_color_space]
+
+#! [ov_image_scale]
+from openvino.preprocess import ResizeAlgorithm
+from openvino.runtime import Layout, Type
+ppp = PrePostProcessor(model)
+input = ppp.input(tensor_name)
+# need to specify H and W dimensions in model, others are not important
+input.model().set_layout(Layout('??HW'))
+# scale to model shape
+input.preprocess().resize(ResizeAlgorithm.RESIZE_LINEAR, 448, 448)
+# and insert operations to the model
+model = ppp.build()
+#! [ov_image_scale]
+
+
+
+model_path = ''
+operation_name = ''
+
+core = Core()
+network = core.ReadNetwork(model_path)
+
+
+#! [mean_scale]
+from openvino.inference_engine import ie_api as ie
```

Comment:
Similar story to importing over and over again. Do not import ie_api as such, this is "low-level" package. Remove imports later and only give one general import note above.
```suggestion
from openvino.inference_engine as ie

model_path = ''
operation_name = ''

core = ie.IECore() # !!! This is old API
network = core.ReadNetwork(model_path)


#! [mean_scale]
```

Reply:
Moved imports to special section at the top of the page.